bigquery/executor.go

```diff
@@ -298,3 +299,78 @@ func GetColumnsFromTable(table string, target *config.Target) (Schema, error) {
 
 	return schema, nil
 }
+
+// LoadSeedFile loads a CSV file to BigQuery as a table.
+func LoadSeedFile(ctx context.Context, seed *fs.SeedFile) error {
+	target, err := seed.GetTarget()
+	if err != nil {
+		return err
+	}
+
+	switch {
+	case target.ProjectID == "":
+		return errors.New("no project ID defined to run query against")
+	case target.DataSet == "":
+		return errors.New("no dataset defined to run query against")
+	}
+
+	client, err := GetClientFor(target.RandExecutionProject())
+	if err != nil {
+		return err
+	}
+
+	f, err := os.Open(seed.Path)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	rs := bigquery.NewReaderSource(f)
+	rs.AllowJaggedRows = false
+	rs.SkipLeadingRows = 1
+	rs.SourceFormat = bigquery.CSV
+
+	if seed.HasSchema() {
+		schema, err := getSeedSchema(seed)
+		if err != nil {
+			return err
+		}
+		rs.Schema = schema
+		rs.AutoDetect = false
+	} else {
+		rs.AutoDetect = true
+	}
+
+	dataset := client.DatasetInProject(target.ProjectID, target.DataSet)
+	loader := dataset.Table(seed.Name).LoaderFrom(rs)
+	loader.WriteDisposition = bigquery.WriteTruncate // Replace table content
+
+	job, err := loader.Run(ctx)
+	if err != nil {
```

Comment:
How will this behave if the target table at `seed.Name` already exists?

Reply:
I'm thinking it should error if the target exists, to avoid accidentally overwriting an existing table

Reply:
It'll replace the content of the target table, but I think it's how it's meant to work, but you highlighted a potentially problem scenario:
- if another seed file targets the same table, we'll [error](https://github.com/monzo/ddbt/pull/13/files#diff-15381eca1abaf2554e8965a20296b696d20d6ea6dad6fbf72c786164581dacf5R211-R213) when we read the files, so we don't race to write to the same table (same check exists for DBT models that targets the same table)
- if another normal DBT model targets the same table, we currently don't detect that, so the seed table could replace the data from the normal DBT model silently, I'll add a commit to check that there are no overlaps between DBT model targets and seed model targets to avoid that

Reply:
Yep exactly 🙈 - sounds good!